.github/ISSUE_TEMPLATE/005-early-design-review.md

```diff
@@ -34,16 +39,19 @@ Further details:
 
 You should also know that...
 
-[please tell us anything you think is relevant to this review]
+{{Please tell us anything else you think is relevant to this review.}}
 
 ------------------------------------------------------------------------------------
 CAREFULLY READ AND DELETE CONTENT BELOW THIS LINE BEFORE SUBMITTING
 
 Please preview the issue and check that the links work before submitting.
 
-In particular:
-* if anything links to a URL which requires authentication (e.g. Google document), please make sure anyone with the link can access the document. We would prefer public documents though, since we work in the open.
+Use links to content rather than pasting text into this issue.  Issues are ephemeral and most of the material we are asking for has long term value.
```

Comment:
I feel like the paragraph before this and the paragraph after should be next to each other. Maybe this one belongs first?